leapp/models/fields/__init__.py

```diff
@@ -108,7 +108,9 @@ def from_initialization(self, source, name, target):
         :type target: Instance of a Model derived class
         :return: None
         """
-        source_value = source.get(name, self._default)
+        # if base class has a field like mutable collection -> copy it, otherwise
+        # the base class and the child class may share the same object
```

Comment:
This comment is just wrong - This is about the instantiation of the Model during the runtime. This problem with the inheritance just showed more drastically a problem with the model defaults.

```python
class SomeModel(Model):
    topic = WhateverTopic
    items = fields.List(fields.String, default=[])
```
has the same problem

Try it like this:
```python
model1 = SomeModel()
model2 = SomeModel()
model1.items.append('woot')
assert model1.items is model2.items
```
The code you see here is the initialization of the runtime values. This has nothing to do anymore with inheritance.